WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAdDestinationSettingViewModel.swift

```diff
@@ -79,15 +126,15 @@ private extension BlazeAdDestinationSettingViewModel {
         enum parameterCharactersLimit {
             static let plural = NSLocalizedString(
                 "blazeAdDestinationSettingVieModel.parameterCharactersLimit.plural",
-                value: "%d characters remaining",
-                comment: "Blaze Ad Destination: Plural form for characters limit label %d will be replaced by a number. " +
+                value: "%1$@ characters remaining",
+                comment: "Blaze Ad Destination: Plural form for characters limit label. %1$@ will be replaced by a number. " +
                 "Read like: 10 characters remaining"
             )
 
             static let singular = NSLocalizedString(
                 "blazeAdDestinationSettingVieModel.parameterCharactersLimit.singular",
-                value: "%d character remaining",
-                comment: "Blaze Ad Destination: Singular form for characters limit label %d will be replaced by a number. " +
+                value: "%1$@ character remaining",
```

Comment:
Similar comment as above about the parameter type.

Reply:
handled in https://github.com/woocommerce/woocommerce-ios/pull/11642/commits/34b30ce01ae95c9a64f1cbecaa53399ba2a3df23